app/src/main/java/com/x8bit/bitwarden/data/auth/datasource/network/model/AuthRequestsResponseJson.kt

```diff
@@ -28,35 +30,43 @@ data class AuthRequestsResponseJson(
      * @param requestApproved Whether this request was approved.
      * @param originUrl The origin URL of this auth request.
      */
+    @OptIn(ExperimentalSerializationApi::class)
     @Serializable
     data class AuthRequest(
         @SerialName("id")
         val id: String,
 
         @SerialName("publicKey")
+        @JsonNames("publickey")
```

Comment:
Can we not do this and just use the interceptor to apply the correct naming.

Reply:
Removed all `@JsonNames` annotations. 👍